### PR TITLE
M120 public

### PR DESCRIPTION
**Pull request by HinTak**
The code changes from m119 to m120 are all obvious, most of them actually mentioned in the top-level release notes. So almost all the changes in the tests are obvious too, just skipping tests involving withdrawn APIs. I don't think those need revisiting, as they are all explicitly mentioned as withdrawn in the release note. Whether we want to emulate them is another matter.

However, somehow there are two new segfaults. I think they are indirectly related / dependent on those withdrawn ones, so they needs re-visiting, either enable them or mark them as indirectly withdrawn.

So this should pass CI, but I'd say those two segfaults need to be either properly skip as withdrawn or re-enabled, before this is merged.

### CODE REVIEW

**Comment by HinTak** [comment]
The segfaults were caused by the withdrawn GL-related API, and actually GrBackendSemaphore only got SK_API status with m120, and GL-backend with this wiil never work. so having the GL-based methods earlier was a mistake. That said, apparently it should be possible if vulkan build is enabled, and should work if the GrBackendSemaphore is initialized with a Vulkan backend: #219 .

@kyamagu I think this is missing a README.m120 but code-wise okay to go, as m121 should be out soon. I'll get on with a README summary soon. Not sure if we should include #219 in this?

**Comment by kyamagu** [comment]
@HinTak Thanks for all of the work! I think it is okay to exclude #219 in the current release as long as the situation is appropriately documented. Vulkan-related APIs are somewhat involved and perhaps require careful testing, which would take too much time to include in this release

File: README.md
[the remark this answers is not in the document]
Comment (HinTak) [reply README.md:1]: Relocated notes to relnotes directory.

**Comment by HinTak** [comment]
As of now,
2072 passed, 108 skipped, 13 xfailed

Compared to v87.5
2171 , 27, 8

**Comment by HinTak** [comment]
Damn - it is "RTL" languages in the release note!

**Pull request merged**